pkgs/development/python-modules/apache-beam/default.nix

```diff
@@ -53,6 +54,15 @@ buildPythonPackage rec {
     sha256 = "sha256-0S7Dj6PMSbZkEAY6ZLUpKVfe/tFxsq60TTAFj0Qhtv0=";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "fix-for-dill-0.3.6.patch";
+      url = "https://github.com/apache/beam/commit/7e014435b816015d21cc07f3f6c80809f3d8023d.patch";
```

Comment:
Do you intend to upstream that patch? It is not on any of upstreams branches yet.